PyInstaller/utils/hooks/__init__.py

```diff
@@ -643,7 +644,11 @@ def seen(p, m={{}}):
                         if onerror is not None:
                             onerror(name)
                         else:
-                            raise
+                            traceback.print_exc(file=sys.stderr)
+                            print(
+                                "collect_submodules: failed to import '{{}}'!"
+                                .format(name), file=sys.stderr
+                            )
```

Comment:
No need to forcefully use `format` everywhere. (Esp. if the format string is wrong ;-)
```suggestion
                            print("collect_submodules: failed to import",
                                  repr(name), file=sys.stderr)
```
I admit this is missing the exclamation mark
```suggestion
                            print("collect_submodules: failed to import %r!" %
                                  name, file=sys.stderr)
```

Reply:
Applied the second suggestion. Thanks!